LICENSE

```diff
@@ -1,57 +1,23 @@
-The Code Project Open License (CPOL) 1.02
+The MIT License (MIT)
 
-Preamble
+Copyright (c) Dapplo and Contributors
```

Comment:
This seems to have been copy/pasted from Dapplo ;)

Reply:
Damn, I was hoping you would over see it, so I have all rights... no just kidding.
Let me correct that...